src/components/Layout.tsx

```diff
@@ -18,7 +17,11 @@ export function Layout({ children }: LayoutProps) {
         </Grid>
         <Sidebar />
         <Grid item bgcolor={grey['100']} padding={4} xs>
-          {children}
+          <Stack spacing={3}>
```

Comment:
Are you sure wen need a nested Outlet in a `Stack` component? If you need padding you use within the components already presents

Reply:
The outlet was needed to make it possible for the Router to render the children routes. With the createBrowserRouters method it is advised to use this.
Should I avoid using nested routes and create them all on the same level to avoid this?

Reply:
My question was about the need of a `Stack` component there